### DISCUSSION TITLE
updating alpine from 2 to 3 breaks anything

### DISCUSSION THREAD

**Discussion by lordisp** [alpinejs/alpine#1714] [1. Help] [answered]
It seams I'm not alone with that issue. I simply updated Alpine like `zaster` reportet on `stackoverflow`

https://stackoverflow.com/questions/68124910/alpine-js-how-can-i-upgrade

Any toggles, dropdowns, modals stops working.

What did we wrong?

**Comment by danddanddand** [comment, marked as answer]
Did you go through everything in the upgrade guide? https://alpinejs.dev/upgrade-guide

**Reply by lordisp** [reply]
yes I did but I found a comment from @calebporzio that [livewire is not supported in V3](https://github.com/alpinejs/alpine/discussions/1426)

**Reply by SimoTod** [reply]
You need to updated livewire (next post on the thread you linked)

**Reply by lordisp** [reply]
yeah just saw it. Also on the docs. I run the latest version `"livewire/livewire": "^v2.5.1"`

If I add the `cdn` it works fine.  I also tried to completely remove AP and re-install it. Same effect. Rolling back to V2, everything works fine.

**Reply by SimoTod** [reply]
If you are packing your own bundle the important bits are: https://alpinejs.dev/upgrade-guide#need-to-call-alpine-start
Alpine needs to start after livewire (in general you would defer the script)

**Reply by lordisp** [reply]
Thanks @SimoTod that was the missing bit! My bad; I probably overflow that part.
